test/setLocale.js

```diff
@@ -23,4 +23,12 @@ describe('Custom locale', () => {
     const locale = require('../src/locale').default;
     expect(locale.string.email).to.equal('Invalid email');
   });
+
+  it('should not allow prototype pollution', () => {
+    const payload = JSON.parse('{"proto":{"polluted":"Yes! Its Polluted"}}');
```

Comment:
Can we change this to:

`const payload = JSON.parse('{"__proto__":{"polluted":"Yes! Its Polluted"}}');`